doc/manual/manual.tex

```diff
@@ -10316,9 +10302,9 @@ \subsubsection{Material models}
 starting point for implementations of new material models.
 
 Older versions of \aspect{} used to have individual functions like viscosity()
-instead of the evaluate() function discussed above. They are now a deprecated
```

Comment:
While you're there, can you also mark up "viscosity()" and "evaluate()" as code?